OAuth/ResourceOwner/DropboxResourceOwner.php

```diff
@@ -12,35 +12,61 @@
 namespace HWI\Bundle\OAuthBundle\OAuth\ResourceOwner;
 
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use HWI\Bundle\OAuthBundle\Security\Core\Authentication\Token\OAuthToken;
+use Buzz\Message\RequestInterface as HttpRequestInterface;
 
 /**
- * DropboxResourceOwner.
+ * DropboxResourceOwner
  *
  * @author Jamie Sutherland<me@jamiesutherland.com>
  */
 class DropboxResourceOwner extends GenericOAuth2ResourceOwner
 {
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
      */
     protected $paths = array(
-        'identifier' => 'uid',
-        'nickname' => 'email',
-        'realname' => 'display_name',
-        'email' => 'email',
+        'identifier' => 'account_id',
+        'nickname'   => 'email',
+        'realname'   => 'display_name',
+        'email'      => 'email',
     );
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
+     */
+    public function getUserInformation(array $accessToken, array $extraParameters = array())
+    {
+        if ($this->options['use_bearer_authorization']) {
```

Comment:
This should be set to `true` by default, cause all v2 endpoints expect that header, also URL is wrong, you should use:
https://www.dropbox.com/developers/documentation/http/documentation#users-get_current_account
And with that URL you don't need to pass content type anymore as well the account id in body.